run_nextflow_mip.sh

```diff
@@ -38,11 +38,11 @@ module load Java/1.8.0_60
 if [ \$? -eq 0 ]; then
     echo "Nextflow done."
 
-    echo "Running Nextflow clean"
-    /hpc/diaggen/software/tools/nextflow clean -f -k -q
```

Comment:
clean command not needed anymore?

Reply:
I believe cleaning via nextflow is replaced with the find command on line 42.

Reply:
check. that explains both my comments I guess :+1: